### DISCUSSION TITLE
Custom labels for get_topic_tree

### DISCUSSION THREAD

**Discussion by clfkenny** [MaartenGr/BERTopic#2122] [Ideas]
Hi there, I noticed the `visualize_hierarchy` method has a `custom_labels` parameter. Is there anything similar to this for the `get_topic_tree` method? I use the LLM generated labels as the custom labels so the hierarchy tree is much easier to interpret. Thanks!

**Comment by MaartenGr** [comment]
That has not been implemented since this visualization was quite different from all the others. Having said that, it would definitely be reasonable to add this at some point. I unfortunately have limited time to work on this at the moment but I'll make sure to convert this to an issue and label it as a "good first issue". That way, other users/developers can help out by creating a PR.
